server/app/core/config.py

```
@@ -68,6 +68,17 @@ class Settings(BaseSettings):
     VAPI_SERVER_SECRET: str = "your_server_secret_here"
     VAPI_SERVER_TIMEOUT_SECONDS: int = 20
     
+    # Telegram Bot Settings
+    ENABLE_TELEGRAM_BOT: bool = True
+    TELEGRAM_BOT_TOKEN: str = "8336316829:AAHpQxyWThm2EEtYNb0j-EJk4Ka32dFgMw0"
```

Ruff (0.14.7)

Possible hardcoded password assigned to: "TELEGRAM_BOT_TOKEN"

(S105)